### PR TITLE
upstream `spacelift`

### PR DESCRIPTION
**Pull request by Benbentwo**
## what
* Updated Spacelift Component to latest
* Updated README with new example

## why
* Upstreams

### CODE REVIEW

**Review by milldr — APPROVED** [review]

File: modules/spacelift/README.md [hunk 1]
```diff
@@ -25,72 +25,94 @@ components:
...
           - spacelift-defaults
+      settings:
+        spacelift:
+          policies_by_id_enabled:
+            # This component also creates this policy so this is omitted prior to the first apply
+            # then added so it's consistent with all admin stacks.
+            - trigger-administrative-policy
       vars:
         enabled: true
-        # Use context_filters to split up admin stack management
```
Comment (nitrocode) [inline modules/spacelift/README.md:81]: Why was this removed? This is a pretty helpful example and we use it in infra-live.
